wremnants/CardTool.py

```diff
@@ -577,7 +583,17 @@ def writeForProcess(self, h, proc, syst, check_systs=True):
             systInfo = self.systematics[syst]
             procDict = self.datagroups.getDatagroups()
             hnom = procDict[proc].hists[self.nominalName]
+            #logger.debug(f"{proc}: {syst}: {h.axes.name}")
+            if systInfo["scaleHist"] != None:
```

Comment:
The hist is being scaled by another histogram? What's the use case?

Reply:
No, this is just a scaling by a constant, currently I need it for tests, to assess whether scaling the histogram directly rather than the sigma encoded in the card is equivalent. For very small variations I suspect that I need to inflate the histogram, or the likelihood may see a shallow direction for that nuisance if I have a small histogram variation and I further call it 1/N of a sigma